.github/workflows/release.yaml

```diff
@@ -114,15 +80,44 @@ jobs:
           SENTRY_ORG: "nodefactory"
           SENTRY_PROJECT: "chainguardian"
           SENTRY_SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
-          
-      
+
+      - name: Generate changelog
+        id: changelog
+        uses: heinrichreimer/github-changelog-generator-action@v2.2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issues: "false"
+          issuesWoLabels: "false"
+          pullRequests: "true"
+          prWoLabels: "true"
+          author: "true"
+          usernamesAsGithubLogins: "true"
+          compareLink: "true"
+          filterByMilestone: "false"
+          unreleased: "false"
+          sinceTag: "${{ needs.tag.outputs.previous_tag }}"
+          maxIssues: "0"
+          stripGeneratorNotice: "true"
+
+      - name: Create Release
```

Comment:
This cannot be after `name: Build/release Electron app` since we wouldn't have release artifacts then. Also this job is executed on every os, which means we would create release 3 times. You should leave everything as it was but add generate changelog in "tag" job so the flow would be following:
Check and Tag:
1. check if new tag and tag
2. create changelog
3. create release
4. (rollback on failure)

Publish:
1. yarn install
2. build and upload release

So tldr:
Just add generate changelog and get previous version to tag job